src/site/content/en/blog/text-fragments/index.md

```diff
@@ -0,0 +1,444 @@
+Additionally, navigations originating from a different origin than the destination
+will require the navigation to take place in a
+[`noopener`](https://html.spec.whatwg.org/multipage/links.html#link-type-noopener) context,
+such that the destination page is known to be sufficiently isolated.
+Text fragment directives are only applied to the main frame.
+This means that text will not be searched inside iframes,
+and iframe navigation will not invoke a text fragment.
+
+### Privacy
+
+It is important that implementations of the Text Fragments specification
+do not leak whether a text fragment was found on a page or not.
+While element fragments are fully under the control of the original page author,
+text fragments can be created by anyone.
+Remember how in my example above there was no way to link to the
+*ECMAScript Modules in Web Workers* heading, since the `<h1>` did not have an `id`,
+but how anyone, including me, could just link to anywhere by carefully crafting the text fragment?
+
+Imagine I ran an evil ad network `evil-ads.example.com`.
+Further imagine that in one of my ad iframes I dynamically created a hidden cross-origin iframe
+to `dating.example.com` with a Text Fragment URL
+<code>dating.example.com<mark class="highlight-line highlight-line-active">#:~:text=Log%20Out</mark></code>
+once the user interacts with the ad.
+If the text "Log Out" is found, I know the victim is currently logged in to `dating.example.com`,
+which I could use for user profiling.
+Since a naive Text Fragments implementation might decide
+that a successful match should cause a focus switch,
+on `evil-ads.example.com` I could listen for the `blur` event and thus know when a match occurred.
+In Chrome, we have implemented Text Fragments in such a way that the above scenario cannot happen.
+
+Another attack might be to exploit network traffic based on scroll position.
+Assume I had access to network traffic logs of my victim, like as the admin of a company intranet.
+Now imagine there existed a long human resources document *What to Do If You Suffer From…*
+and then a list of conditions like *burn out*, *anxiety*, etc.
+I could place a tracking pixel next to each item on the list.
+If I then determine that loading the document temporally co-occurs
+with the loading of the tracking pixel next to, say, the *burn out* item,
+I can then, as the intranet admin, determine that an employee has clicked through
+on a text fragment link with `:~:text=burn%20out`
+that the employee may have assumed was confidential and not visible to anyone.
+Since this example is somewhat contrived to begin with
+and since its exploitation requires *very* specific preconditions to be met,
+the Chrome security team evaluated the risk of implementing scroll on navigation to be manageable.
+Other user agents may decide to show a manual scroll UI element instead.
+
+For sites that still wish to opt-out, we have proposed a
```

Comment:
I hate to throw a monkey wrench into this at the 11th hour. It looks like the opt-out code laded back in mid-April. No one told us or even bothered to [close the ticket until last Thursday](https://bugs.chromium.org/p/chromium/issues/detail?id=1047900). (Here's a link showing that this [merged to 83](https://storage.googleapis.com/chromium-find-releases-static/942.html#942995187c607e433af35e114a325c4ca2576d37) some time ago.)

cc: @kaycebasques, @petele

Reply:
Sorry about the confusion (I've reopened that bug). The thing that landed and was merged back to M83 was the origin trial - the DocumentPolicy remains unshiped. So I think this text is still correct as-is.